CodeBase/TwoSum.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * PROBLEM: Two Sum in JavaScript language
+ * AUTHOR: Shubham0523
+ **/
+
+function twoSum(nums, target) {
+    let obj = {}; // Create an empty object to store the indices of numbers
+    for (let i = 0; i < nums.length; i++) {
+        let n = nums[i];
+        if (obj[target - n] >= 0) {
+            return [obj[target - n], i];
+        } else {
+            obj[n] = i; // Store the index of the current number
+        }
+    }; 
+}
```

Comment:
Please call this method `twoSum` and add a `console.log` for printing the result so that we can run the code and verify also.

Reply:
And please do support us by giving staring this repository